perfrunbook/debug_hw_perf.md

```diff
@@ -230,6 +231,11 @@ the samples directly, you can use the [Arm SPE Parser](https://gitlab.arm.com/te
 
 Be aware that SPE produces a large of volume of data (many GBs) if the sampling period is low and you collect the data over a long time.
 
+### Capturing cache coherence issues
+The `perf c2c` command can be used to analyze cache coherence issues and false sharing in multi-core systems.
```

Comment:
Please add info that this uses SPE on Graviton, which is only available on metal instances, and requires loading the SPE driver (`apt-get install linux-modules-extra-$(uname -r) && modprobe arm_spe_pmu`).

Reply:
Do we need to add for AL as well?

Reply:
The existing text does have instructions to load the SPE driver. So, not adding those again in this PR.